README.md

```diff
@@ -185,6 +185,29 @@ async def purge(ctx: commands.Context, tz_info: str, military_time: bool):
 ```
 </details>
 
+<details><summary><b>Asset handler</b></summary>
```

Comment:
Can you place this section under `Additional Functions`, then perhaps change the title to `Additional Functions/Features`.